GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
+        if let colorValue = keyColorPair.1?.rgbColor?.hexValue {
```

Comment:
No I'm fine with your version, I was just loving how we could make amazing stuff like this with Swift  :wink: 

But if you had to change, I'm not sure using `map` is worth it there. Why not simply:
```
for colorName in colorsList.allKeys {
  if let colorValue = colorsList.colorWithKey(colorName)?.rgbColor?.hexValue {
    ...
  }
}
```